lib/roger/partition/consumer.ex

```diff
@@ -337,7 +339,14 @@ defmodule Roger.Partition.Consumer do
   defp consume(queue, state) do
     queue_name = Queue.make_name(state.partition_id, queue.type)
     # FIXME: do something with stats?
-    {:ok, _stats} = AMQP.Queue.declare(queue.channel, queue_name, durable: true)
+    {:ok, _stats} =
+      AMQP.Queue.declare(queue.channel, queue_name,
+        durable: true,
```

Comment:
I'm missing the option: `persistent: true`, which is required for quorum-queues (but may be set implicitely)